### ISSUE TITLE
Can't capture scrolled elements

### ISSUE THREAD

**Issue by dushyanttessact** [tsayen/dom-to-image#289]
### Use case: description, code  

[jsfiddle](https://jsfiddle.net/dushyant42/42cv6nLa/1/)

scroll down and then capture the screen, check screenshot it will only show the top elements and not the scrolled ones.

**Comment by andomain** [comment]
I am also having this issue

**Comment by MSCAU** [comment]
You can turn off the overflow, run DomToImage, then turn it back on again. If that's too distracting to the user, append a clone of your chosen DIV with overflow: visible at the bottom of the page. I do this on https://beatjosh.com/budgets/FY2019/CTH (click the SNAPSHOT button in the top right toolbar).
Not ideal, but perhaps good enough.

**Comment by kalandher** [comment]
is there a solution for this? i wanted to capture only the scrolled content. for e.g if there are 100 rows in a table and I scrolled at 50th and do capture is should get from 50th row but now its going back to top. Even turnoff overflow is not helping here as it always scroll top to before capture and my scrolled content is not captured.

**Comment by HenryLin0914** [comment]
+1

**Comment by moazzam-ali** [comment]
Yeah, I also want to just capture what's visible. I want it to retain the scroll progress on the image for all the vertically or horizontally scrollable elements.

Any solution or any alternative library or something!!
